client/notices/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+import i18n from 'lib/mixins/i18n';
+import notices from 'notices';
+
+const NoticesGlobal = () => {
+	function callSuccessNotice() {
+		notices.success( i18n.translate( 'This is a global success notice' ) );
+	}
+
+	function callErrorNotice() {
+		notices.error( i18n.translate( 'This is a global error notice' ) );
+	}
+
+	function callInfoNotice() {
+		notices.info( i18n.translate( 'This is a global info notice' ) );
+	}
+
+	function callWarningNotice() {
+		notices.warning( i18n.translate( 'This is a global warning notice' ) );
+	}
+
+	return (
```

Comment:
Super minor syntax you may not have considered (not necessary to change): if you move those function declarations out of the component, you can omit the `return` statement here altogether:

```js
const NoticesExample = () => (
  <div>
    { /* the rest of the component */ }
  </div>
);
```